src/java.base/share/classes/java/text/MessageFormat.java

```diff
@@ -307,12 +307,12 @@
  * will be the final result of the parsing.  For example,
  * <blockquote><pre>
  * MessageFormat mf = new MessageFormat("{0,number,#.##}, {0,number,#.#}");
- * Object[] objs = {new Double(3.1415)};
+ * Object[] objs = {Double.valueOf(3.1415)};
  * String result = mf.format( objs );
  * // result now equals "3.14, 3.1"
  * objs = null;
```

Comment:
Although this is not related to this fix, this line is not needed (and is confusing as an example snippet). I'd remove this.